### DISCUSSION TITLE
My phone is restarting for no reason

### DISCUSSION THREAD

**Discussion by Appfanatic456** [community/community#50230] [Mobile]
### Select Topic Area

Bug

### Body

So, I plugged my phone into a telephone handset to charge it and after a minute my phone started to glitch out and then it just showed the apple logo. I think the handset downloaded some software on my phone or something. Does anyone have any ideas on how to fix this?

**Comment by Iron-Ham** [comment]
Hi @Appfanatic456

This doesn't seem to be related to the GitHub Mobile application, but I hope you find a solution to this problem. Maybe try contacting the handset manufacturer?

**Reply by mocrow-3sytdo-ceZsym** [reply]
`
[
![image](https://user-images.githubusercontent.com/128235336/226619039-d2ceabfd-cc27-431a-a5e0-15f74829b7cd.png)]()`
